src/store/board-service/reducer.ts

```diff
@@ -0,0 +1,19 @@
+import { Reducer } from "redux";
+
+interface BoardState {
```

Comment:
move to src/utils/types/reducers/board-reducer.interface.ts and rename the interface to IBoardReducer. your initial state should be an Array of users interface. cause a table it's always the collection of data.